fsproxy/src/main/java/no/unit/identityservice/fsproxy/model/FsCourse.java

```diff
@@ -0,0 +1,40 @@
+package no.unit.identityservice.fsproxy.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+import nva.commons.core.JacocoGenerated;
+
+public class FsCourse {
+
+    @JsonProperty("undervisning")
+    private final FsUndervisning undervisning;
```

Comment:
Mish mash of Norwegian and english terms. Use the `JsonProperty` so that you can match the the norwegian term to an english-named field